week4/src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from 'react'
+import GlobalStyle from "./styles/GlobalStyle";
+import { RouterProvider } from "react-router-dom";
+import { router } from "./Router";
+import { ThemeProvider } from "styled-components";
+import theme from "./styles/theme"
+
+function App() {
+  return (
+    <>
+      <GlobalStyle />
+      <ThemeProvider theme={theme}>
+      <RouterProvider router={router} />
+      </ThemeProvider>
+
+    </>
```

Comment:
다음과 같이 해주면 빈 태그를 사용하지 않아도 되겠죠!!
```suggestion
      <ThemeProvider theme={theme}>
        <GlobalStyle />
        <RouterProvider router={router} />
      </ThemeProvider>
```

Reply:
> 다음과 같이 해주면 빈 태그를 사용하지 않아도 되겠죠!!

빈 태그를 사용하면 문제가 되는 부분이 있나요??

Reply:
> 다음과 같이 해주면 빈 태그를 사용하지 않아도 되겠죠!!

빈 태그를 사용하면 문제가 되는 부분이 있나요??